frontend/src/components/regions/UserDashboard/UserDetailedStats.js

```diff
@@ -7,10 +7,10 @@ import {
   TableRow,
   Typography,
 } from "@material-ui/core";
-
+import { cardColors } from "../../../colors";
 import { PieChart, Pie, Cell, Tooltip } from "recharts";
 
-const COLORS = ["#0088FE", "#00C49F", "#FFBB28", "#FF8042"];
+const [...COLORS] = Object.values(cardColors);
```

Comment:
Take a look at how the colours are accessed. Do you think just grabbing the colours by index makes sense? Should we not be writing code that explicittly grabs the colour by status?  (Line 79)

Right now, if I were to randomly rearrange the colours in the colours.js file then the colours in this file will start to mean differnt things. So this file needs to know about the order of the colours in a completely different file. This is brittle.